cmd/juju/controller/kill.go

```diff
@@ -104,9 +104,17 @@ func (c *killCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
 	}
 	store := c.ClientStore()
-	if !c.assumeYes {
+
+	skipConfirm, skipErr := jujucmd.CheckSkipConfirmEnvVar()
+	if skipErr != nil {
+		return errors.Trace(skipErr)
+	}
+	if c.assumeYes {
+		fmt.Fprint(ctx.Stdout, "WARNING: '-y'/'--yes' flags a deprecated and will be removed in JUJU 3.1\n")
+	}
+	if !(c.assumeYes || c.assumeNoPrompt || skipConfirm) {
```

Comment:
Don't forget to drop `skipConfirm` here as well, like in destroy-controller

Reply:
Done